.github/workflows/build-package.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and Publish Docker Image
+
+on:
+  push:
+    branches: ['main', 'feature/upgrades']
```

Comment:
```suggestion
    branches: ['main']
    pull_request:
```

Reply:
yes. testing for now 👀

Reply:
but you already have a PR open? I suppose you'll want an if clause for the actual push below

Reply:
oh I should make this draft again. I didn't realize there was a PR already